oat-sa/tao-core
# Feature/tr 885 rtl modal

**Related to**
https://oat-sa.atlassian.net/browse/TR-885

**Summary**
Added styles for the modal popup to respect RTL direction

**Testing**
1) update dependency for tao-core like `"oat-sa/tao-core": "dev-feature/TR-885_RTL_modal as 48.21.0",`
2) create a test with warnings
3) create a user with the Arabic language by default
4) set test delivery to the user
5) Run test runner and check popup for the user

**Depends on** 
https://github.com/oat-sa/tao-core-ui-fe/pull/335

2/2
